src/pages/spawn/spawnHost/EditSpawnHostModal.tsx

```diff
@@ -178,6 +182,7 @@ export const EditSpawnHostModal: React.FC<EditSpawnHostModalProps> = ({
                 })
               }
               value={editSpawnHostState.instanceType}
+              disabled={!canEditInstanceType}
```

Comment:
can we conditionally render a tooltip explaining why this component is disabled?